src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1198,7 +1202,17 @@ private ObjectNode prepareBidderParameters(ExtRequestPrebid prebid, String bidde
 
         final long startTime = clock.millis();
 
-        return httpBidderRequester.requestBids(bidder, bidderRequest, timeout, requestHeaders, debugEnabledForBidder)
+        final BidRequest bidRequest = bidderRequest.getBidRequest();
+        final BidRequest modifiedBidRequest = bidderMediaTypeProcessor != null
+                ? bidderMediaTypeProcessor.process(bidRequest, resolvedBidderName, auctionContext)
+                : bidRequest;
```

Comment:
Lets make BidderMediaTypeProcessor an interface with basic(current implementation) and noOp one, like https://github.com/prebid/prebid-server-java/blob/78b12ff6fd5c62ec70b36d4da2aacfd91891e00b/src/main/java/org/prebid/server/auction/categorymapping/NoOpCategoryMappingService.java#L11

and choose implementation via Spring configuration.